lib/queries/useAuthorityMappingRules/useAuthorityMappingRules.js

```diff
@@ -1,18 +1,19 @@
 import { useQuery } from 'react-query';
 
-import {
-  useOkapiKy,
-  useNamespace,
-} from '@folio/stripes/core';
+import { useNamespace } from '@folio/stripes/core';
 
-const useAuthorityMappingRules = () => {
-  const ky = useOkapiKy();
+import { useTenantKy } from '../../temp/hooks/useTenantKy';
+
+const useAuthorityMappingRules = ({ tenantId, enabled = true }) => {
```

Comment:
If there is a place where this function is invoked without args the page will be crashed. Let's utilize the default value {}. Please check other functions too.

Reply:
There are only two places and I updated them both, but I added the default arguments just in case, thanks